pkg/xds/generator/inbound_proxy_generator.go

```diff
@@ -14,6 +14,7 @@ import (
 	envoy_names "github.com/kumahq/kuma/pkg/xds/envoy/names"
 	v3 "github.com/kumahq/kuma/pkg/xds/envoy/routes/v3"
 	"github.com/kumahq/kuma/pkg/xds/envoy/tags"
+	xds_tls "github.com/kumahq/kuma/pkg/xds/envoy/tls/v3"
```

Comment:
we should not import individual Envoy versions in generator. `KumaALPNProtocols` should be common to all versions. I'd suggest moving `KumaALPNProtocols` out of `pkg/xds/envoy/tls/v3` into `pkg/xds/envoy/tls`